Automattic/wp-calypso
# Improve theming capabilities for Sidebar and Masterbar

#### Changes proposed in this Pull Request

* Improve theming capabilities for Sidebar and Masterbar
* Update overflow-gradient Mixin
* Update long-content-fade Mixin

#### Testing instructions

* Build the styles, everything should work as expected.
* Navigate the site using the Default (Classic Blue) color scheme. The Sidebar and Masterbar should look identical color wise to what you see in production.
